.github/workflows/unit-test.yml

```diff
@@ -0,0 +1,22 @@
+name: Unit Test
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        node: [8, 10, 12]
```

Comment:
Only difference you can not specify latest stable node version, but the feature is coming: https://github.com/actions/setup-node/pull/58